src/components/button/button_group/button_group.tsx

```diff
@@ -166,13 +171,24 @@ export const OuiButtonGroup: FunctionComponent<Props> = ({
     );
   }
 
+  // Compressed style can't support `vertical` orientation because compressed button group has a fixed height
```

Comment:
I don't get why this would be a blocker. A variation of `height: auto` with a fixed-width would be enough to bypass this. no?

Reply:
@KrooshalUX what's your opinion of a vertical compressed button should look like?

Reply:
@KrooshalUX It would be nice to have UX suggestion on a `vertical` + `compressed` button

Reply:
Docs on `compressed`: https://oui.opensearch.org/1.2/#/forms/compressed-forms

I think the idea is to have any sort of spacing smaller than uncompressed